tasks/nmg_transform.py

```diff
@@ -0,0 +1,47 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+class NgmTransForm(BaseSalesforceApiTask):
```

Comment:
This doesn't need to be a `BaseSalesforceApiTask`; it doesn't talk to the API. Looks like `BaseTask` should be fine here.